packages/preview/anatomy/0.1.0/lib.typ

```diff
@@ -35,7 +35,10 @@
     let stroke_width = 0.5pt
     let description-size = (1 + 2 / 3) / 10 * 1em
 
-    block(width: 100%, above: 0pt, {
+    block(
+      width: 100%,
+      above: description-size,
+      below: description-size, {
```

Comment:
Add extra spacing at the top and the end of the `metrics`, so users can have the preferable spacing without surrounding `metrics` by another `block`.